 /* Ethernet frame types according to RFC 2516 */
 #define ETH_PPPOE_DISCOVERY 0x8863
--- a/include/linux/ppp_defs.h
+++ b/include/linux/ppp_defs.h
@@ -38,6 +38,8 @@
 #ifndef _PPP_DEFS_H_
 #define _PPP_DEFS_H_
 
+#include <sys/time.h>	/* time_t */
+
 /*
  * The basic PPP frame.
  */
